modules/nsq/src/main/scala/com/snowplowanalytics/snowplow/enrich/nsq/GcsClient.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2023 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.snowplow.enrich.nsq
+
+import java.net.URI
+
+import cats.implicits._
+import cats.effect.{Blocker, ConcurrentEffect, ContextShift, Timer}
+
+import fs2.Stream
+
+import blobstore.gcs.GcsStore
+import blobstore.url.Url
+
+import com.google.cloud.storage.StorageOptions
+import com.google.cloud.BaseServiceException
+
+import com.snowplowanalytics.snowplow.enrich.common.fs2.io.Clients.{Client, RetryableFailure}
+
+object GcsClient {
```

Comment:
Is there an argument this should live in the common-fs2 module instead of repeating it in the stream-specific modules?  common-fs2 already [has dependencies on the s3 and gcs clients](https://github.com/snowplow/enrich/blob/3.8.0/project/Dependencies.scala#L280-L281)

Reply:
My mistake, I was looking at the wrong dependencies.